prow/plugins/label/label.go

```diff
@@ -213,6 +223,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	mID, err := strconv.Atoi(maintainersID)
+	if err != nil {
+		log.WithError(err).Errorf("Couldn't convert the maintainers team ID parameter to int %v", maintainersID)
+	}
+	milestoneMaintainers, err := gc.ListTeamMembers(mID)
+	maintainersMap := map[string]bool{}
+	if err == nil {
+		for _, person := range milestoneMaintainers {
+			maintainersMap[person.Login] = true
+		}
+	} else {
+		log.WithError(err).Errorf("Failed to list the teammembers for the milestone maintainers team")
```

Comment:
Does it make sense to try to proceed? I think you can just quit here and propagate up the error.

Reply:
there's the `sigMatches` section below that checks for `/sig <>` type commands.  I don't think I can/should return early

Reply:
Ok